hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_matcher.py

```diff
@@ -78,6 +100,7 @@ def lambda_handler(event, context):
                 )
                 signal_id = metadata["id"]
 
+                # TODO: Add source (threatexchange) tags to match record
```

Comment:
Is this still a TODO? Looks like source is included in the match record on line 109

Reply:
Looks like that's the threat exchange ID and not the tags. One more lookup is needed to get the tags.